src/main/java/edu/harvard/iq/dataverse/search/AdvancedSearchPage.java

```diff
@@ -66,322 +72,246 @@ public void init() {
-                queryStrings.add(constructQuery(dsfType.getSolrField().getNameSearchable(), dsfType.getSearchValue()));
-            } else if (dsfType.getListValues() != null && !dsfType.getListValues().isEmpty()) {
-                List<String> listQueryStrings = new ArrayList<>();
-                for (String value : dsfType.getListValues()) {
-                    listQueryStrings.add(dsfType.getSolrField().getNameSearchable() + ":" + "\"" + value + "\"");
-                }
-                queryStrings.add(constructQuery(listQueryStrings, false));
-            }
-        }
-        if (StringUtils.isNotBlank(dsPublicationDate)) {
-            queryStrings.add(constructQuery(SearchFields.DATASET_PUBLICATION_DATE, dsPublicationDate));
-        }
-        if (StringUtils.isNotBlank(dsPersistentId)) {
-            queryStrings.add(constructQuery(SearchFields.DATASET_PERSISTENT_ID, dsPersistentId));
-        }
-        return constructQuery(queryStrings, true);
+    // -------------------- PRIVATE --------------------
 
+    private void mapDataversesAndFilesBlocks() {
+        dataversesSearchBlock = new SearchBlock("dataverses",
+                BundleUtil.getStringFromBundle("advanced.search.header.dataverses"), constructDataversesSearchFields());
+
+        filesSearchBlock = new SearchBlock("files",
+                BundleUtil.getStringFromBundle("advanced.search.header.files"),
+                constructFilesSearchFields());
     }
 
-    private String constructDataverseQuery() {
-        List<String> queryStrings = new ArrayList<>();
-        if (StringUtils.isNotBlank(dvFieldName)) {
-            queryStrings.add(constructQuery(SearchFields.DATAVERSE_NAME, dvFieldName));
-        }
-        if (StringUtils.isNotBlank(dvFieldAlias)) {
-            queryStrings.add(constructQuery(SearchFields.DATAVERSE_ALIAS, dvFieldAlias));
-        }
+    private void mapAllMetadataBlocks(List<DatasetFieldType> metadataFieldList) {
+        for (MetadataBlock mdb : metadataBlocks) {
 
-        if (StringUtils.isNotBlank(dvFieldAffiliation)) {
-            queryStrings.add(constructQuery(SearchFields.DATAVERSE_AFFILIATION, dvFieldAffiliation));
-        }
+            List<SearchField> searchFields = mapMetadataBlockFieldsToSearchFields(metadataFieldList, mdb);
 
-        if (StringUtils.isNotBlank(dvFieldDescription)) {
-            queryStrings.add(constructQuery(SearchFields.DATAVERSE_DESCRIPTION, dvFieldDescription));
+            metadataSearchBlocks.add(new SearchBlock(mdb.getName(), mdb.getLocaleDisplayName(), searchFields));
         }
+        addExtraFieldsToCitationMetadataBlock();
+    }
 
-        if (dvFieldSubject != null && !dvFieldSubject.isEmpty()) {
-            List<String> listQueryStrings = new ArrayList<>();
-            for (String value : dvFieldSubject) {
-                listQueryStrings.add(SearchFields.DATAVERSE_SUBJECT + ":" + "\"" + value + "\"");
```

Comment:
Have you looked how queries was translated before? This looks like some values were quoted. In new code there is no support for it.